_includes/serverless/serverless-preview-warning.md

```diff
@@ -0,0 +1,2 @@
+{: .warning}
+FeatureBase Serverless is currently in preview and is not meant to be used in production workloads. It is only available through FeatureBase Cloud.
```

Comment:
Make this more succinct by adding something like:

```
{: .warning}
FeatureBase Serverless should not be used for production workloads as it is currently in preview.
```
Also, as this is only going to appear in `cloud-db` then there's no requirement to state the functionality is available only for FB cloud